Sources/AccountSDKIOSWeb/User/User.swift

```diff
@@ -74,8 +74,8 @@ public class User: UserProtocol {
      - parameter redirectURI: where to redirect the user after the session has been created
      - parameter completion: callback that receives the URL or an error in case of failure
      */
-    public func webSessionURL(clientId: String, redirectURI: String, completion: @escaping HTTPResultHandler<URL>) {
-        client.schibstedAccountAPI.sessionExchange(for: self, clientId: clientId, redirectURI: redirectURI) { result in
+    public func webSessionURL(clientId: String, state: String? = nil, redirectURI: String, completion: @escaping HTTPResultHandler<URL>) {
```

Comment:
Please document state parameter :)

Reply:
Also, there are already unit tests for both classes User and SchibstedAccountApi. We need to update or add a new test to be sure that everything works as desired.